src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactor/ProofOfPossessionResult.php

```diff
@@ -91,7 +132,7 @@ public function didOtpVerificationFail()
     /**
      * @return boolean
      */
-    public function didProofOfPossessionFail()
+    public function didProofOfPossessionCommandFail()
```

Comment:
Is it relevant to the consumer of the service that it was in fact the execution of a command that failed? Imo the consumer is merely asking the possession to be proven using whatever method the service sees fit.